src/20231205_Multi_OnboardNativeUSDCToAaveV3Markets/AaveV3Arbitrum_OnboardNativeUSDCToAaveV3Markets_20231205.sol

```diff
@@ -0,0 +1,110 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3Arbitrum, AaveV3ArbitrumAssets, AaveV3ArbitrumEModes} from 'aave-address-book/AaveV3Arbitrum.sol';
+import {AaveV3PayloadArbitrum} from 'aave-helpers/v3-config-engine/AaveV3PayloadArbitrum.sol';
+import {EngineFlags} from 'aave-helpers/v3-config-engine/EngineFlags.sol';
+import {IAaveV3ConfigEngine} from 'aave-helpers/v3-config-engine/IAaveV3ConfigEngine.sol';
+import {IV3RateStrategyFactory} from 'aave-helpers/v3-config-engine/IV3RateStrategyFactory.sol';
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+import {SafeERC20} from 'solidity-utils/contracts/oz-common/SafeERC20.sol';
+
+/**
+ * @title Onboard Native USDC to Aave V3 Markets
+ * @author Aave Chan Initiative and @EzR3aL
+ * - Snapshot: No snapshot for Direct-to-AIP
+ * - Discussion: https://governance.aave.com/t/arfc-onboard-native-usdc-to-aave-v3-markets/15720
+ */
+contract AaveV3Arbitrum_OnboardNativeUSDCToAaveV3Markets_20231205 is AaveV3PayloadArbitrum {
+  using SafeERC20 for IERC20;
+
+  function rateStrategiesUpdates()
+    public
+    pure
+    override
+    returns (IAaveV3ConfigEngine.RateStrategyUpdate[] memory)
+  {
+    IAaveV3ConfigEngine.RateStrategyUpdate[]
+      memory rateStrategies = new IAaveV3ConfigEngine.RateStrategyUpdate[](1);
+    rateStrategies[0] = IAaveV3ConfigEngine.RateStrategyUpdate({
+      asset: AaveV3ArbitrumAssets.USDC_UNDERLYING,
+      params: IV3RateStrategyFactory.RateStrategyParams({
+        optimalUsageRatio: EngineFlags.KEEP_CURRENT,
+        baseVariableBorrowRate: EngineFlags.KEEP_CURRENT,
+        variableRateSlope1: _bpsToRay(7_00),
+        variableRateSlope2: _bpsToRay(80_00),
+        stableRateSlope1: EngineFlags.KEEP_CURRENT,
+        stableRateSlope2: EngineFlags.KEEP_CURRENT,
+        baseStableRateOffset: EngineFlags.KEEP_CURRENT,
+        stableRateExcessOffset: EngineFlags.KEEP_CURRENT,
+        optimalStableToTotalDebtRatio: EngineFlags.KEEP_CURRENT
+      })
+    });
+
+    return rateStrategies;
+  }
+
+  function capsUpdates() public pure override returns (IAaveV3ConfigEngine.CapsUpdate[] memory) {
+    IAaveV3ConfigEngine.CapsUpdate[] memory capsUpdate = new IAaveV3ConfigEngine.CapsUpdate[](1);
+
+    capsUpdate[0] = IAaveV3ConfigEngine.CapsUpdate({
+      asset: AaveV3ArbitrumAssets.USDC_UNDERLYING,
+      supplyCap: 26_000_000,
+      borrowCap: 24_000_000
+    });
+
+    return capsUpdate;
+  }
+
+  function collateralsUpdates()
+    public
+    pure
+    override
+    returns (IAaveV3ConfigEngine.CollateralUpdate[] memory)
+  {
+    IAaveV3ConfigEngine.CollateralUpdate[]
+      memory collateralUpdate = new IAaveV3ConfigEngine.CollateralUpdate[](2);
+
+    collateralUpdate[0] = IAaveV3ConfigEngine.CollateralUpdate({
+      asset: AaveV3ArbitrumAssets.USDC_UNDERLYING,
+      ltv: 77_00,
+      liqThreshold: 80_00,
+      liqBonus: 5_00,
```

Comment:
I think the current liquidation bonus already is 5% so we can use `KEEP_CURRENT` here